nuxeo/values.yaml

```diff
@@ -9,6 +9,9 @@ image:
 ##   - `replicaCount` API nodes: handling HTTP requests, with workers and stream processing disabled
 ##   - `workerCount` worker nodes: not handling HTTP requests, with workers and stream processing enabled
 architecture: singleNode
+strategy: 
+  # Can be replaced by Recreate. Useful to sequence a pod upgrade and avoid any pvc issue with two overlapping instances.
+  type: RollingUpdate
```

Comment:
I put the default so we don't need to check the existence of `Values.strategy` and then `Values.strategy.type`

Reply:
And it's a good practice to make the default value explicit.